index.html

```diff
@@ -10559,20 +10562,21 @@ <h3>User Prompt Handler</h3>
  <li><p>If the <a>user prompt handler</a> is null, set the <a>user prompt
  handler</a> to an empty map.
 
- <li><p>For each <var>request key</var> → <var>request handler</var>
+ <li><p>For each <var>request prompt type</var> → <var>request handler</var>
  in <var>requested prompt handler</var>:
 
   <ol>
-   <li><p>Set <a>user prompt handler</a>[<var>key</var>] to <var>value</var>.
-
+   <li><p>Set <a>user prompt handler</a>[<var>request prompt type</var>]
+   to <var>request handler</var>.
   </ol>
 
 </ol>
 
 <p>To <dfn>serialize the user prompt handler</dfn>:
 
 <ol class=algorithm>
- <li><p>If the <a>user prompt handler</a> is null, return null.
+ <li><p>If the <a>user prompt handler</a> is null, return
```

Comment:
This is a behaviour change, right? I don't disagree with making this change if it aligns with more implementations, but we shouldn't sneak it in in a bugfix PR. Also you need to update the caller to not handle null if we're always returning a string.

Reply:
This is the change that we discussed on Matrix. If there is no `unhandledPromptBehavior` capability specified we won't return anything. But at the moment both Firefox and Chrome return the default which is `dismiss and notify`. 

If we are sure that we absolutely don't want to return the default value if the capability wasn't set when calling the new session command, I'm happy to revert that and add a test to check that more strict behavior (which aligns to what Safari is doing).

Reply:
I think the point is that you should split that change into a separate PR and make sure we get input from other vendors, since it's not just an editorial change.

Reply:
Ok. I'm going to remove this changeset then and will file a new issue to discuss if we should return the capability or not for the default case.